### PR TITLE
Issue 322

### PR DESCRIPTION
**Pull request by Squawk09**
Program now displays when the account will be negative at the end of the month.  Tests are also included.

### CODE REVIEW

File: Src/MoneyFox.Shared/App.cs [hunk 1]
```diff
@@ -30,8 +28,7 @@ private void RegisterDependencies()
             Mvx.RegisterType<IDatabaseManager, DatabaseManager>();
             Mvx.RegisterSingleton<IPasswordStorage>(new PasswordStorage(Mvx.Resolve<IProtectedData>()));
             Mvx.RegisterType(() => new Session());
-            Mvx.RegisterSingleton<IMvxTextProvider>(new MvxResxTextProvider(Strings.ResourceManager));
```
Comment (NPadrutt) [inline Src/MoneyFox.Shared/App.cs]: Why did you delete this registration?

File: Src/MoneyFox.Shared/Manager/EndOfMonthManager.cs [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+        public void AssignToAccounts()
+        {
+            foreach(Account x in accountRepository.GetList())
+            {
+                DeterminEndThroughAccounts(x);
+            }
+        }
+
+
+        /*
```
Comment (NPadrutt) [inline Src/MoneyFox.Shared/Manager/EndOfMonthManager.cs]: make the comments with /// this generates a documentation header (for example in the data access classes are some of them).

File: Src/MoneyFox.Windows/Converter/FinalValueConverter.cs [hunk 3]
```diff
@@ -0,0 +1,12 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace MoneyFox.Windows.Converter
```
Comment (NPadrutt) [inline Src/MoneyFox.Windows/Converter/FinalValueConverter.cs]: is this class needed? seems pretty empty?

**Comment by Squawk09** [comment]
Ok, yes all of these changes make sense.  I think having the account background turn red is a good idea.  The only issue I see is with removing the string warning.  From the user's perspective, they will not intuitively understand  why the account suddenly turned red. Should there be maybe a shorter string or something else to also convey why the color changed?

Thanks!

**Comment by NPadrutt** [comment]
sure, a short string may make sense. Let's just try it and see how it works out =)

**Comment by Squawk09** [comment]
Ok, sounds good. Ill revise it tonight

**Comment by NPadrutt** [comment]
Let me know when I shall review it again :)

**Comment by Squawk09** [comment]
Ok, 

I have included a coloring warning as well as integrate your edits.  Please let me know if you think anything else should be changed.

I also found a couple bugs to report. 

Thanks!

**Comment by NPadrutt** [comment]
great =) I'll revise it tomorrow. and feel free to greate issues for all the bugs you've found! :)
